dmutils/filters.py

```diff
@@ -45,3 +46,30 @@ def format_links(text):
         return formatted_text
     else:
         return text
+
+
+def timesince(dt, default="just now"):
+    """
+    Returns string representing "time since" e.g.
+    3 days ago, 5 hours ago etc.
+    """
```

Comment:
We have tests for these filters in tests/test_filters.py

You should add some more for this new one.